.github/workflows/deploy.yml

```
@@ -0,0 +1,211 @@
+      uses: aws-actions/configure-aws-credentials@v4
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: ${{ env.AWS_REGION }}
+
+    - name: Create deployment package
+      run: |
+        mkdir -p package
+        uv export --no-dev --format requirements-txt > requirements.txt
+        pip install -r requirements.txt -t package/
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)